exercises/concept/secrets/.docs/instructions.md

```diff
@@ -0,0 +1,55 @@
+# Instructions
+
+Your has just sent you a message with an important secret.
+Not wanting to make it easy for others to read it, the message has been encrypted using a series of bit manipulations.
+You will need to write a program to help decrypt the message.
+
+## 1. Shift back the bits
+
+The first step in decrypting the message is to undo the shifting from the encryption process by shifting the bits back to the left.
+The number of places to shift can vary between messages.
+
+Implement the method `Secrets.shift_back` that takes a value and the number of places to shift left.
```

Comment:
This doesnt match how the method is tested in the spec files. Should be `Secrets::shift_back`

Reply:
With this one, I ended deciding to update the tests, stub and exemplar files to match.